doc/source/options.rst

```diff
@@ -37,7 +37,7 @@ namespace:
 - :func:`~pandas.option_context` - execute a codeblock with a set of options
   that revert to prior settings after execution.
 
-**Note:** developers can check out pandas/core/config.py for more info.
+**Note:** Developers can check out ``pandas/core/config.py`` for more info.
```

Comment:
you can put this link in: https://github.com/pandas-dev/pandas/blob/master/pandas/core/config.py